opennlp-tools/src/test/java/opennlp/tools/namefind/NameFinderMETest.java

```diff
@@ -137,7 +134,8 @@ public void testNameFinderWithTypes() throws Exception {
     params.put(TrainingParameters.CUTOFF_PARAM, Integer.toString(1));
 
     TokenNameFinderModel nameFinderModel = NameFinderME.train("en", TYPE, sampleStream,
-        params, (byte[]) null, Collections.<String, Object>emptyMap());
+        params, TokenNameFinderFactory.create(null, null, Collections.emptyMap(),
+        TokenNameFinderFactory.instantiateSequenceCodec(null)));
```

Comment:
Probably better to just specify the codec by name or just create it, e.g. new BioCodec()